integration/sawtooth_integration/tests/test_py_executor.rs

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
We should follow this pattern for these kinds of tests (not quite integration but not quite unit): https://doc.rust-lang.org/stable/rust-by-example/testing/integration_testing.html. You should use `[dev-dependencies]` to specify libraries that are only required by testing (intkey here).

Reply:
Thanks for the link.